migrations/v5.js

```diff
@@ -0,0 +1,62 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('adapt-contrib-accordion - v4.0.0 > v5.0.0', async () => {
+  let accordions;
+
+  whereFromPlugin('adapt-contrib-accordion - from v4.0.0', { name: 'adapt-contrib-accordion', version: '<=5.0.0' });
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = content.filter(({ _component }) => _component === 'accordion');
+    if (accordions) return true;
+  });
+
+  /**
+   * * Add JSON field to component and set attribute.
+   */
+  mutateContent('adapt-contrib-accordion - update accordion._supportedLayout attribute to full-width', async () => {
+    accordions.forEach(accordion => {
+      /**
+      * ? Would this potentially break the layout of any course being migrated?
```

Comment:
I would say there's a good chance of that. `half-width` is still a valid layout. Perhaps only set if `_supportedLayout` does not have a valid value?

Reply:
Done